netutils/config/conversion.py

```diff
@@ -0,0 +1,36 @@
+"""Configuraiton conversion methods for different network operating systems."""
+# pylint: disable=no-member,super-with-arguments,invalid-overridden-method,raise-missing-from,invalid-overridden-method,inconsistent-return-statements,super-with-arguments,redefined-argument-from-local,no-else-break,useless-super-delegation,too-many-lines
+
+from typing import Generator, List
+
+
+def paloalto_panos_brace_to_set(config: Generator[str, None, None]) -> List[str]:
```

Comment:
We should have tests for these. You should follow the pattern of glob shown here: https://github.com/networktocode/netutils/blob/develop/tests/unit/test_compliance.py#L19-L27

Reply:
Test added